core-bundle/src/Doctrine/Backup/BackupManager.php

```diff
@@ -27,22 +27,32 @@ class BackupManager
 
     private Connection $connection;
     private DumperInterface $dumper;
-    private string $backupDir;
+    private VirtualFilesystemInterface $backupsStorage;
```

Comment:
Just a minor nitpick: I think it is called "backup storage" instead of "backups storage", even though there might be multiple backups in the same storage. 🤓

Reply:
It's because autowiring would name it `backupsStorage` because it's named `backups` and not `backup`. Same as `files` :)